ops_preflight.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_runtime_dependency('thor', '~> 0.19.1')
+  gem.add_runtime_dependency('mime-types', '>= 2.99', '< 4.0')
```

Comment:
This version spec will work with what we have in our production app currently (2.99).